invenio_openaire/tasks.py

```diff
@@ -28,51 +28,55 @@
 
 from celery import shared_task
 from invenio_db import db
-from invenio_pidstore.models import PersistentIdentifier, PIDStatus
+from invenio_pidstore.errors import PIDDoesNotExistError
+from invenio_pidstore.resolver import Resolver
 from invenio_records.api import Record
 
-from invenio_openaire.loaders import RemoteFundRefLoader, RemoteOAIRELoader
+from .loaders import LocalFundRefLoader, LocalOAIRELoader, \
+    RemoteFundRefLoader, RemoteOAIRELoader
+from .minters import funder_minter, grant_minter
 
 
 @shared_task(ignore_result=True)
-def harvest_fundref(loader=None):
```

Comment:
Yes. The object will have to be pickled to be sent through RabbitMQ to a Celery, and we don't support pickle as a serialiser (Celery is removing support for it as it's a security issue). We only support JSON, msgpack and YAML by default. I.e basically only simple dicts should be sent to Celery.